src/BenchmarksDriver/Program.cs

```diff
@@ -156,6 +156,14 @@ public static int Main(string[] args)
                 "-w|--webHost",
                 "WebHost (Kestrel or HttpSys). Default is Kestrel.",
                 CommandOptionType.SingleValue);
+            var aspnetCoreVersionOption = app.Option("--aspnetCoreVersion",
+                "ASP.NET Core version (2.0.0 or 2.1.0-*).  Default is 2.1.0-*.", CommandOptionType.SingleValue);
+            var runtimeFrameworkVersionOption = app.Option("--runtimeFrameworkVersion",
```

Comment:
I think this can be condensed down to a single version with values like "2.0.0", "2.0.1", and "latest".  If the version is "latest", we just use whatever is set in the `Benchmarks` branch we clone and build.  If the version is say "2.0.0", we should set the environment variables `KOREBUILD_DOTNET_SHARED_RUNTIME_VERSION` and `KOREBUILD_DOTNET_VERSION` to control the runtime and CLI installed by KoreBuild, and also the MSBuild variables `BenchmarksAspNetCoreVersion`, `BenchmarksRuntimeFrameworkVersion` (and possibly `BenchmarksNETStandardImplicitPackageVersion` and `BenchmarksNETStandardLibraryNETFrameworkVersion` if they're still relevant).